ethcore/src/tx_filter.rs

```diff
@@ -60,17 +60,21 @@ impl TransactionFilter {
 	/// Check if transaction is allowed at given block.
 	pub fn transaction_allowed<C: BlockInfo + CallContract>(&self, parent_hash: &H256, transaction: &SignedTransaction, client: &C) -> bool {
 		let mut cache = self.permission_cache.lock();
-
+		let mut to: Address = Address::new();
 		let tx_type = match transaction.action {
```

Comment:
I'd prefer a tuple to be returned from `match` statement instead of modifying `to`.